fl4health/clients/nnunet_client.py

```diff
@@ -624,23 +630,6 @@ def empty_cache(self) -> None:
         elif self.device.type == "mps":
             torch.mps.empty_cache()
 
-    def update_after_step(self, step: int, current_round: Optional[int] = None) -> None:
```

Comment:
So in this code, regardless of whether we were training by epochs or by steps in the client, the lr scheduler was being provided with self.total_steps as part of its update. Now, we won't be providing that regardless of the training scheme. 

I don't know the inner workings of the PolyLRScheduler, but at the very least, we're not preserving the warning that Shawn had here. Perhaps we should fold this in by overriding the `update_lr_schedulers` function?

Reply:
The reason I had to remove the warning is that the LR Scheduler has attributes (ie max_steps) that are not shared by all LR Scheduler. So we couldn't have general logic to raise an exception if total steps exceeds max_steps. But I agree I can add this check in by overriding the method